.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+Thank you for sending a PR !
```

Comment:
This is nice :) but could be included as a comment so it's invisible when publishing the PR. As in

```html
<!-- comments! -->
```